tests/wpt/metadata/webdriver/tests/execute_script/collections.py.ini

```diff
@@ -14,9 +14,6 @@
   [test_html_collection]
     expected: FAIL
```

Comment:
It looks like array-like objects that are not arrays aren't working yet?

Reply:
Thanks for noticing that! Yeah, it seems that I forgot to handle the case where the array's elements are WebElements.